changelog.md

```diff
@@ -1,5 +1,8 @@
 # Change log
 
+## Unreleased
+Config for Custom translations- remoteTranslations
```

Comment:
```suggestion
Config for retrieving translations from a new Languages API - `useTranslationsAPI`. This API is not yet available but adding a feature toggle in advance for development.
```

Reply:
Don't be afraid to elaborate the changes for external audiences that don't have our knowledge - Kibble is a public repo.